src/mixins/color-mixin.js

```diff
@@ -166,7 +166,11 @@ export default function colorMixin(_chart) {
     const range = _chart.colors().range()
     const middleColor = range[Math.floor(range.length / 2)]
 
-    return _colors(_colorAccessor.call(this, data, index)) || middleColor
+    const value = _colorAccessor.call(this, data, index)
+
+    return typeof value === "string"
```

Comment:
Probably safe to do this for all data types, since hashing function handles numbers as well as strings, but just have it turned off for strings atm